portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/Forms.testcase

```diff
@@ -1084,8 +1084,8 @@ definition {
 			rowEntry = "Created Form Name");
 	}
 
-	@ignore = "true"
 	@description = "This is a use case for LPS-98073."
+	@ignore = "true"
```

Comment:
@alinedoleron, why didn't you squash this commit with the third one? Let's commit less putting together very related commits, ok?